app/root/root.css

```diff
@@ -1550,7 +1550,7 @@ code .comment {
 }
 
 @media (max-width: 800px) {
-  .hide-on-mobile {
+  .hide-on-mobile.hide-on-mobile {
```

Comment:
this is one of the rare situations where i think !important actually makes sense--this way only works until someone makes a three-deep selector that sets display somewhere else.

Reply:
Done